js/main.js

```diff
@@ -1,2 +1,44 @@
 /* global data */
 /* exported data */
+var $title = document.querySelector('#note-title');
+var $urlInput = document.querySelector('#url');
+var $notes = document.querySelector('#entry-notes');
+var $submit = document.querySelector('form');
+
+function setImage(event) {
+
+  document.querySelector('img').setAttribute('src', $urlInput.value);
+}
+
+$urlInput.addEventListener('blur', setImage);
+
+function resetForm(event) {
+  data.entries.push({
+    title: $title.value,
+    photoURL: $urlInput.value,
+    notes: $notes.value
+  });
+  data.nextEntryId += 1;
+
+  $title.value = '';
+  $urlInput.value = '';
+  $notes.value = '';
+  document.querySelector('img').setAttribute('src', 'images/placeholder-image-square.jpg');
+  event.preventDefault();
```

Comment:
I generally prefer that `event.preventDefault()` is the first line of any function it needs to be a part of, for the sake of readability